### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Moh88CS**
Ensure all links are functional as you read through the document. Comment if anything should be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,59 @@
...
+
+You can find information about the structure and design of our app in our [Design Document](https://tarheels.live/comp523teami/architecture-diagram/)!
+
+## Accessing and Running FanzPlay
+
+For detailed information on how to run the app, please refer to both the User Manual and the Adminstrator Manual on our app website in the section of [Documentation Plan](https://tarheels.live/comp523teami/documentation-plan/).
+
+You can also refer to your [Handoff Plan](https://tarheels.live/comp523teami/handoff-plan/) which includes helpful videos on how to access and run the app.
+
+## Implementation details
```
Comment (mayather) [inline README.md]: This is a bit nitpicky but would we be able to capitalize "details"?

Comment (Moh88CS) [reply README.md]: Should be fixed now

File: README.md [hunk 2]
```diff
@@ -1,31 +1,59 @@
...
+The testing is done inside the [test folder](https://github.com/stefanodongowski/FanzPlay/tree/stage/test) in this repo.
 
 ## Linting
-Before merging your branch, be sure to lint the code by running 
-`npm run lint` and `npm run prettier:write` in the terminal. 
 
+Before merging your branch, be sure to lint the code by running
+`npm run lint` and `npm run prettier:write` in the terminal. This will properly format your code and lint to maintain code readability and consistency.
+
+## Maintabality
```
Comment (mayather) [inline README.md]: I think maintainability may be spelled wrong?

Comment (Moh88CS) [reply README.md]: You are correct. Fixed now

**Review by mayather — APPROVED** [review]
LGTM!

**Pull request merged**